docs/resources/agent_to_server.md

```diff
@@ -8,19 +8,19 @@ description: |-
 
 # thousandeyes_agent_to_server (Resource)
 
+This resource allows you to create and configure an agent-to-server test. This test type measures network performance as seen from ThousandEyes agent(s) towards a remote server. For more information about agent-to-server tests, see [Agent-to-Server Tests]([https://docs.thousandeyes.com/product-documentation/internet-and-wan-monitoring/tests#agent-to-agent-test](https://docs.thousandeyes.com/product-documentation/internet-and-wan-monitoring/tests#agent-to-server-test).
 
-
-
+## Example Usage
 
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
 ### Required
 
-- `agents` (Block Set, Min: 1) Agents to use (see [below for nested schema](#nestedblock--agents))
```

Comment:
Everything under ` <!-- schema generated by tfplugindocs -->` will be overwritten by `go generate`. These descriptions will need to be changed in [schemas.go](https://github.com/thousandeyes/terraform-provider-thousandeyes/blob/496382e168f660e21f5a640f2632658af043506e/thousandeyes/schemas.go).